jupyterhub_moss/spawner.py

```diff
@@ -194,17 +193,34 @@ def _slurm_info_resources(self, slurm_info_out):
         help="Singularity command to use for starting jupyter server in container",
     ).tag(config=True)
 
-    FORM_TEMPLATE = Environment(
-        loader=FileSystemLoader(TEMPLATE_PATH),
-        autoescape=False,
-        trim_blocks=True,
-        lstrip_blocks=True,
-    ).get_template("option_form.html")
-
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.options_form = self.create_options_form
 
+    @functools.cached_property
+    def __option_form_template(self):
+        """jinja2 template
+
+        Templates look-up is similar to JupyterHub:
+        Use 'templates/...' to extend and fall-back to embedded templates
+        """
+        template_path = local_path("templates")
+        loader = ChoiceLoader(
+            [
+                PrefixLoader({"templates": FileSystemLoader(template_path)}),
+                FileSystemLoader(
+                    list(self.user.settings["template_path"]) + [template_path]
```

Comment:
`self.user.settings["template_path"]`: This is the way I found to retrieve the template paths set for JupyterHub :shrug: